### DISCUSSION TITLE
Can you support TdEngine

### DISCUSSION THREAD

**Discussion by changyuxin** [hhyo/Archery#2795] [General]
support TdEngine?

**Comment by LeoQuote** [comment]
你好, 本项目是一个开源项目, 你可以自由提功能需求, 但其实比提功能需求更加务实, 有价值的是实现这个功能, 本项目开发文档如下, 希望对你有帮助:

* https://github.com/hhyo/Archery/wiki/development
* https://github.com/hhyo/Archery/wiki/notifier

期待你的pr, 谢谢~
